Changes to PaymentMethod documentation (#181)
killbill/slate: source/includes/_payment-method.md

Basic operations to retrieve, list, search and delete payment methods.

> Note that the creation of a payment method needs to be done on the Account level and the endpoint listed under [Account resource](https://killbill.github.io/slate/#account-add-a-payment-method) must be used for this. The `pluginName` attribute identifies the payment plugin that will be used by the system when this payment method is used for a payment. 

Reviewer: ```suggestion
Note that the creation of a payment method needs to be done at the Account level using the [Add a Payment Method](https://killbill.github.io/slate/#account-add-a-payment-method) endpoint. The `pluginName` attribute identifies the payment plugin that will be used by the system when this payment method is used for a payment. 
```

Author: Hi @reshmabidikar made this change and added the commit.
